dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -299,6 +299,38 @@ def check_docker_compose_version(verbose: bool):
         )
 
 
+def check_docker_context(verbose: bool):
+    """
+    Checks whether Docker is using the expected context
+    :param verbose: print commands when running
+    """
+    expected_docker_context = "default"
+    response = run_command(
+        ["docker", "info", "--format", "{{json .ClientInfo.Context}}"],
+        verbose=verbose,
+        no_output_dump_on_exception=False,
+        text=True,
+        capture_output=True,
+    )
+    if response.returncode != 0:
+        get_console().print(
+            '[error]Could not check for Docker context.[/]\n'
+            '[warning]Please make sure Docker is installed and running properly.[/]'
+        )
+
+    used_docker_context = response.stdout.strip().replace('"', '')
+
+    if used_docker_context == expected_docker_context:
+        get_console().print(f'[success]Good Docker context used: {used_docker_context}.[/]')
+    else:
+        get_console().print(
+            f'[error]Docker is not using the default context, used context is: {used_docker_context}[/]\n'
+            f'[warning]Please make sure Docker is using the {expected_docker_context} context.[/]\n'
+            f'[warning]You can try switching contexts by running: "docker context use '
+            f'{expected_docker_context}"[/]'
+        )
+
```

Comment:
We should `sys.exit(1)` here. We know it makes no sense to continue. Save breeze the pain of trying fruitlessly to continue and kill it mercifully and immediately.

Reply:
I was torn a bit about this, because it depends on the flow of `breeze`, for example, when you run `breeze build-image`, continuing past this is not making sense. But, when running just `breeze` to connect to a container, it works even with the wrong context, it's just using an old image instead of an updated one.
BUT
Running with an outdated image can lead to strange behaviors and errors that will just fail the CI, so I've added the `sys.exit(1)`.

Thanks for this comment :)